bin/internal/update_dart_sdk.ps1

```diff
@@ -43,8 +43,15 @@ if (Test-Path $dartSdkPath) {
 }
 New-Item $dartSdkPath -force -type directory | Out-Null
 $dartSdkZip = "$cachePath\$dartZipName"
-Import-Module BitsTransfer
-Start-BitsTransfer -Source $dartSdkUrl -Destination $dartSdkZip
+
+Try {
+    Import-Module BitsTransfer
+    Start-BitsTransfer -Source $dartSdkUrl -Destination $dartSdkZip
+}
+Catch {
+    Write-Host "Downloading the Dart SDK using BITS failed, falling back to WebRequest"
```

Comment:
Can we change this to:
`Downloading the Dart SDK using the BITS service failed, retrying with WebRequest...`?